src/jit/codegenlegacy.cpp

```diff
@@ -13013,7 +13013,7 @@ void CodeGen::genCodeForBBlist()
 
             case BBJ_CALLFINALLY:
 
-#if defined(_TARGET_X86_)
+#if defined(_TARGET_X86_) && !FEATURE_EH_FUNCLETS
 
```

Comment:
This change is introduced to fix build failure. ``lvaShadowSPslotsVar`` is not available when ``FEATURE_EH_FUNCLETS`` is set.